src/PositionDescriptor.sol

```diff
@@ -26,14 +26,27 @@ contract PositionDescriptor is IPositionDescriptor {
     address private constant WBTC = 0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599;
 
     address public immutable wrappedNative;
-    string public nativeCurrencyLabel;
+    bytes32 private immutable nativeCurrencyLabelBytes;
```

Comment:
why is this private?

Reply:
can make public?

Reply:
daniel suggested we make it private since we already have a public getter. for nativeCurrencyLabel

Reply:
i dont see why it can't just be public tho.. theres no downside